app/src/main/java/com/plcoding/socialnetworktwitch/feature_chat/presentation/message/MessageScreen.kt

```diff
@@ -29,24 +29,30 @@ import com.plcoding.socialnetworktwitch.core.presentation.ui.theme.ProfilePictur
 import com.plcoding.socialnetworktwitch.core.presentation.ui.theme.SpaceMedium
 import com.plcoding.socialnetworktwitch.feature_chat.presentation.message.components.OwnMessage
 import com.plcoding.socialnetworktwitch.feature_chat.presentation.message.components.RemoteMessage
+import com.ramcosta.composedestinations.annotation.Destination
+import com.ramcosta.composedestinations.navigation.DestinationsNavigator
 import kotlinx.coroutines.flow.collect
 import okio.ByteString.Companion.decodeBase64
 import java.nio.charset.Charset
 
-@ExperimentalComposeUiApi
-@ExperimentalCoilApi
+data class MessageScreenNavArgs(
+    val remoteUserId: String,
+    val remoteUsername: String,
+    val encodedRemoteUserProfilePictureUrl: String,
+    val chatId: String?
+)
```

Comment:
I left these classes right beside the screen composables just so you can more easily check them, but I'd move them to their own file 🙂